sonic-net/sonic-mgmt
# [dualtor]: Add MAC move switchover test

### Description of PR


Summary:
Fixes # (issue)

### Type of change





### Back port request

### Approach
#### What is the motivation for this PR?
Add test coverage for scenario where MAC move occurs during switchover

#### How did you do it?
Use kernel signals to pause/resume orchagent and syncd to replicate issue

#### How did you verify/test it?
- Run the test on a dualtor testbed with image version < 20201231.89 and verify the test fails
- Run the test on a dualtor testbed with image versien >= 20201231.89 and verify the test passes

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
